### ISSUE TITLE
DRBoxInputEncoder unexpected behaviour

### ISSUE THREAD

**Issue by LLukas22** [ThalesAleniaSpace/DRBox_keras#8]
Hi,
I dont have a lot of experience regarding Keras or Tensorflow and your Repo helped me quite a lot. I tryed training on my own dataset and the loss was all over the place. After diggin depper it seems that **DRBoxInputEncoder** or **drbox_output_decoder.decode_detections()** dont seem to work properly. 
I would assume that after encoding and decoding i should get boxes similar to the original_labels in my *.csv file.
But my decoded boxes are completly wrong.
**Original:**
![Original](https://user-images.githubusercontent.com/65088241/98285550-02adb180-1fa3-11eb-9093-94b5b5076b52.png)
**Decoded:**
![Decoded](https://user-images.githubusercontent.com/65088241/98285546-fde8fd80-1fa2-11eb-8ed8-c7906275012d.png)

Im using a datagenerator without any transformations and the default parameter from "Ship". 

My test labelfile looks like this:
`images/88243053-01.jpg,71.65723870967743,262.91226768060835,74.20142795698925,17.372033460076047,  6.283185307179586,1
images/88243053-01.jpg,71.62080000000002,306.6019771863118,74.35537204301076,17.372033460076047,  6.283185307179586,1`

Did i miss some parameter or is my labelfile not valide?

**Comment by ppontisso** [comment]
Hello,
Yes you should get similar bounding boxes, before and after decoding.
Your label file look fine, so I don't really know how to help you sorry.

If you find an error in the code, feel free to create a pull request with your improvements.
Thank you
